doc/related_projects.rst

```diff
@@ -1,9 +1,29 @@
+.. _related_projects:
+
 =====================================
 Related Projects
 =====================================
 
 Below is a list of sister-projects, extensions and domain specific packages.
 
+Related Packages
+----------------
+Other packages useful for data analysis and machine learning.
+
+- `Pandas <http://pandas.pydata.org>`_ Tools for working with heterogeneous and
+  columnar data, relational queries, time series and basic statistics.
+
+- `sklearn_pandas <https://github.com/paulgb/sklearn-pandas/>`_ bridge for
+  scikit-learn pipelines and pandas data frame with dedicated transformers.
+
+- `theano <http://deeplearning.net/software/theano/>`_ A CPU/GPU array
+  processing framework geared towards deep learning research.
+
+- `Statsmodel <http://statsmodels.sourceforge.net/>`_ Estimating and analysing
+  statistical models. More focused on statistical tests and less on prediction
+  than scikit-learn.
+
+
```

Comment:
scikit-image is under "domain specific packages" where nltk should also be. http://scikit-learn.org/dev/related_projects.html